docs/source/usage.rst

```diff
@@ -1,65 +1,18 @@
-Usage
-=====
-
-.. _installation:
-
-Installation
-------------
-
-``garak`` is a command-line tool. It's developed in Linux and OSX.
-
-Friendly install instructions are at `<https://docs.garak.ai/garak/llm-scanning-basics/setting-up/installing-garak>`_ .
-The instructions below will work, but you might need to be quite familiar with your OS to use them, because they assume some particular pieces of background knowledge.
-
-Standard quick `pip` install
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-To use garak, first install it using pip:
-
-.. code-block:: console
-
-   pip install garak
-
-
-Install development version with `pip`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The standard pip version of ``garak`` is updated periodically. To get a fresher version, from GitHub, try:
-
-.. code-block:: console
-
-    python3 -m pip install -U git+https://github.com/NVIDIA/garak.git@main
-
-
-For development: clone from `git`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can also clone the source and run ``garak`` directly. This works fine and is recommended for development.
-
-``garak`` has its own dependencies. You can to install ``garak`` in its own Conda environment:
-
-.. code-block:: console
-
-    conda create --name garak "python>=3.10,<=3.12"
-    conda activate garak
-    gh repo clone NVIDIA/garak
-    cd garak
-    python3 -m pip install -r requirements.txt
-
-OK, if that went fine, you're probably good to go!
-
-
-Running garak
--------------
-
+..  headings: = - ^ "
+Getting Started
+===============
 
 The general syntax is:
 
 .. code-block:: console
 
-    garak <options>
+   garak <options>
+
+You must specify at least the model to scan with the ``--model_name`` option.
```

Comment:
While `-n` or `--model_name` are required by most `generators` not all require a model name.
```suggestion
You must specify at least the service to scan with the ``--model_type`` option.
```

Not this also should be corrected in the next section as the terms are inverted:
```
To specify a generator, use the ``--model_type`` and, optionally, the ``--model_name`` options.
Model type specifies a model family/interface; model name specifies the exact model to be used.
The "Intro to generators" section below describes some of the generators supported.
A straightfoward generator family is Hugging Face models; to load one of these, set ``--model_type`` to ``huggingface`` and ``--model_name`` to the model's name on Hub (e.g. "RWKV/rwkv-4-169m-pile").
```